bftengine/cmf/execution_data.cmf

```diff
@@ -6,6 +6,8 @@ Msg RequestInput 1 {
     uint64 flags
     string request
     string signature
+    # 0 - raw on-chain 1 - hash on chain 
+    uint8 requestPersistencyType
```

Comment:
CMF supports `enum`. Let's use that instead.

Reply:
great advice, I didn't know it supports it. thanks!

Reply:
though I guess I'm the first user since it doesn't compile on all machines :) fixed it